### ISSUE TITLE
Accessibility Testing for Sitewide content and IA, VALife static page

### ISSUE THREAD

**Issue by aliyahblackmore** [department-of-veterans-affairs/va.gov-team#50636]
### Product information

- [X] Team name, product name, and feature name have been added to the title of this issue.
- [X] Team label, product label and feature label (if applicable) have been added to this issue.

### Who completed the use of color and color contrast test?

@aliyahblackmore 

### Use of color and color contrast checks

- [X] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [X] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [X] Non-text elements have a 3:1  contrast ratio to their background and to neighboring elements (or better)
- [X] Color is not the only way to distinguish links from other text (eg. links are underlined)
- [X] Any charts, maps, infographics, and tables convey all information without only relying on color
- [X] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")

### How did color testing go?

Using the color settings on my computer, I took a look at the 3 different colorblindness filters (deuteranopia, protanopia, and tritanopia). Everything looked great. 

I also checked on WhoCanUse and it seems that everything passed there as well. I added the color code via "Inspect."  Color contrast ratio surpassed the requirements listed above.  Of course, please let me know if this should've been done differently at all!

### Who completed the axe scans?

@laurwill

### axe checks

- [X] Each page has been scanned using axe (results shared below)
- [ ] axe is integrated into your end-to-end testing

### axe DevTools scan results

We weren't able to use axe DevTools for this test due to being in Citrix with the Drupal preview link.
Brian suggested we run the Lighthouse accessibility scan that's built into Chrome.

Our results are from Lighthouse and everything looked great - there was only one issue with the "Back to top" button (outlined in the next question).

### How did axe testing go?

@briandeconinck the only issue was the "Back to top" button, which we believe is a known issue. 

Attaching a screen capture of what we saw.
![image(2)](https://user-images.githubusercontent.com/102090263/207152503-59e9c33d-2071-4c8a-8306-6081222fee8a.png)

### Who completed the content zoom and reflow test?

@aliyahblackmore 

### Content zoom and reflow checks

- [X] All page elements are readable and usable at 200% zoom
- [X] All page elements are readable and usable at 300% zoom
- [ ] All page elements are readable and usable at 400% zoom

### How did content zoom and reflow testing go?

@briandeconinck - the feedback button at the bottom of the page gets cut off at 400%. There isn't an option to scroll horizontally.

<img width="1194" alt="image(1)" src="https://user-images.githubusercontent.com/102090263/207115911-9576ffb4-a02b-40d6-9526-6ec97bf99202.png">

### Who completed the keyboard navigation test?

@aliyahblackmore 

### Keyboard navigation checks

- [X] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
- [X] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
- [X] All elements under focus have a visible focus indicator
- [X] The order of [Tab] stops made sense and was appropriate for completing tasks

### Do you have any other results to share?

As noted earlier, we used Lighthouse for our testing!

**Comment by RLHecht** [comment]
I believe this can be closed.
